installers/ansible/roles/pgo-operator/tasks/cleanup.yml

```diff
@@ -276,7 +276,7 @@
   file:
     state: absent
     path: "/usr/local/bin/pgo"
-  when: pgo_client_install == "true"
+  when: pgo_client_install | default(False) | bool
```

Comment:
The documentation says it's defaulting `true`? Do we need the `default(False)` here?

Reply:
No i don't think we do. Like the other places, this is set in a defaults yaml so the block wouldn't be used. Fixing.